gensim/models/word2vec.py

```diff
@@ -1116,17 +1116,25 @@ def doesnt_match(self, words):
         dists = dot(vectors, mean)
         return sorted(zip(dists, words))[0][1]
 
-    def __getitem__(self, word):
-        """
-        Return a word's representations in vector space, as a 1D numpy array.
+    def __getitem__(self, words):
 
-        Example::
+      """
+      Return a numpy array of N word representations in vector space, as a N X VEC_LENGTH 2D numpy array.
 
-          >>> trained_model['woman']
-          array([ -1.40128313e-02, ...]
+      Example::
+
+      >>> trained_model[['office','products']]
+        array([ -1.40128313e-02, ...]
+              [ -1.70425311e-03, ...]
+              ...)
+
+      """
+      if isinstance(words, string_types):
+          # allow calls like trained_model['office'], as a shorthand for trained_model[['dog']]
```

Comment:
This is a common, supported use case -- please mention it explicitly in the docstring too. Also replace `'dog'` by `'office'` here, the comment is a little confusing now.